Bump `bigdecimal` & fix failing test

cakebaker opened the pull request:
This PR bumps `bigdecimal` from `0.4.7` to `0.4.8` and fixes a failing test.

RenjiSann commented:
@drinkcat If you want to take a look at the test fix

drinkcat commented:
Thanks for tagging me! ,-)

Huh... 3 things:
 1. The `Display` trait implementation is actually dead code, nothing uses it (apart from the unit test just below)...
 2. When I was looking at it, I never really quite understood what was the intent of this line anyway: `Self::BigDecimal(BigDecimal::new(n * 10, 1)).fmt(f)`, seems like an attempt to get a decimal point in all cases? But is this the right thing to do?
 3. The failing test is inconsistent anyway: `::zero` gets formatted as `0.0`, but `MinusZero` as `-0`?

So... up to you. IMHO you could delete the implementation, or change the test to check for `0` instead.

If we keep the implementation, I'd remove that `n*10,1` thing too (and just do `Self::BigDecimal(x) => x.fmt(f)`).

Oh, and a 4th thing ,-P You could grep for `https://github.com/akubera/bigdecimal-rs/issues/144` in the code and remove the workaround and enable the tests as part of this (but I'm happy to do it if you don't have cycles!).

cakebaker commented:
Changes since last push:
* removed `Display` implementation of `ExtendedBigDecimal`
* fixed TODOs related to `https://github.com/akubera/bigdecimal-rs/issues/144`

drinkcat commented:
LGTM thanks!
